academy/golang-intro/1_install.md

```diff
@@ -0,0 +1,89 @@
+---
+title: "Go Introduction"
+order: 
+description: Get started with Golang
+tag: deep-dive
+---
+
+# Go Introduction
+
+Golang (Go) is an open source, statically typed and compiled programming language introduced by Google and first released in 2012. It has a BSD-style license. Its Git repository is located at [https://go.googlesource.com/go](https://go.googlesource.com/go).
+
+Start with the basics. In this section, you install Go and learn some general things about it.
+
+## Install
+
+Installing Go depends on your machine. See the [Getting Started Page](https://golang.org/doc/install) and follow the instructions to install Go. 
+
+<HighlightBox type="tip">
+
+[gimme](https://github.com/travis-ci/gimme) is a useful script for Go.
+
+</HighlightBox>
+
+## Workflow
+
+Go has a different workflow from most other programming tools. You will work with a single workspace and have all Go code contained in this workspace. The `GOPATH` environment variable specifies the location of your workspace. You can print it by typing `$ go env GOPATH` in your console. For more information about `GOPATH` type `$ go help gopath`. 
+
+A version control system is helpful because you have one workspace. You should use Git for this.
+
+## Hello World!
+
+Let's start with the ever-popular "Hello, World!" program. Let's see, how it looks in Go:
+
+```golang
+package main
+import "fmt"
+func main() {
+    fmt.Printf("Hello, World!\n")
+}
+```
+
+[Test online](https://go.dev/play/p/1u5bSZlh80h)
+
+Every Go program starts with the package declaration. Each package consists of one or more Go source files in a single directory. If the name of the package is `main`, Go creates an executable file.
+
+After that, import the package `fmt`. Note that the package name is the last element of the package's file path. For an example, if you import the package "lib/math", then you use it as "math".
```

Comment:
It is not clear what "After that" refers to. 

Suggestion: "After running the **_Hello, World!_** program, ..."

Confirm please!